src/modelfree/configs/multi/train.py

```diff
@@ -131,6 +180,361 @@ def hyper(train):
         _ = locals()  # quieten flake8 unused variable warning
         del _
 
+    ###############################################################################################
+    # BEGIN CONFIGS FOR DEFENSE EXPERIMENTS
+    ###############################################################################################
+    @multi_train_ex.named_config
+    def hyper_finetune_generic(train):
+        """ *** Needs to be run alongside one of hyper_finetune_defense or dual_defense """
```

Comment:
Having the outer ones call the generics seems good; I think I prefer that to the global variable approach, since it more matches other existing patterns in the code